source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,123 @@
+
+The following sections describe how you can enable Versioned API for
+your MongoDB client and each of the options that you can specify.
+
+Enable Versioned API on a MongoDB Client
+----------------------------------------
+
+To enable the Versioned API, you must specify an API version in the settings
+that you use to create your MongoDB client. Once you instantiate a
+``MongoClient`` instance with a specified Versioned API, all commands you
+run with that client use that version of the Versioned API. If you need to
+run commands not covered by the Versioned API or with a different version of
+it, instantiate a separate client.
+
+The example below shows how you can instantiate a ``MongoClient`` that
+specifies version "1" of the Versioned API and connects to a server by
+performing the following operations:
+
+- Construct a ``ServerApi`` instance using the ``ServerApi.Builder``
+  helper class.
+- Construct a ``MongoClientSettings`` instance using the
+  ``MongoClientSettings.Builder`` class.
+- Specify a server to connect to using a ``ServerAddress`` instance.
+- Instantiate a ``MongoClient`` using the ``MongoClients.create()`` method
+  and pass your ``MongoClientSettings`` instance as a parameter.
+
+.. literalinclude:: /includes/fundamentals/code-snippets/VersionedApiExample.java
+   :start-after: start serverAPIVersion
+   :end-before: end serverAPIVersion
+   :language: java
+   :dedent:
+
+.. warning::
+
+   Make sure that when you specify an API version that you connect to a
+   version of MongoDB server that supports Versioned API. Otherwise, your
+   application may raise an exception when attempting to perform any operation
+   that sends a command to MongoDB. For example, a query to a server could
+   return an exception message that resembles the following:
+
+   .. code-block:: none
+      :copyable: false
+
+      com.mongodb.MongoQueryException: Query failed with error code 9 and error message 'Failed to parse: {...} Unrecognized field 'apiVersion'.' on server
+
+For more information on the methods and classes referenced in this
+section, see the following API documentation:
+
+- :java-docs:`ServerApi <apidocs/mongodb-driver-core/com/mongodb/ServerApi.html>`
+- :java-docs:`ServerApi.Builder <apidocs/mongodb-driver-core/com/mongodb/ServerApi.Builder.html>`
+- :java-docs:`ServerAddress <apidocs/mongodb-driver-core/com/mongodb/ServerAddress.html>`
+- :java-docs:`MongoClientSettings <apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.html>`
+- :java-docs:`MongoClientSettings.Builder <apidocs/mongodb-driver-core/com/mongodb/MongoClientSettings.Builder.html>`
+- :java-docs:`MongoClients.create() <apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html#create(com.mongodb.MongoClientSettings)>`
+- :java-docs:`MongoClient <apidocs/mongodb-driver-sync/com/mongodb/client/MongoClient.html>`
+
+Additional Options
+------------------
+
+You can set additional options to enable or disable behavior related to
```

Comment:
Nit: There didn't seem to be options prior to this, so saying additional options may make the reader think there was.
```suggestion
You can set options to enable or disable behavior related to
```

Reply:
Thanks for pointing this out. I'll update the section title too. I think I had considered the "version" to be an option and "strict" and "deprecationErrors" to be additional. Since "version" is required, I can see how referring to it as an option could be confusing.